avinda_shamal/test/unit/test_hf_get_item.py

```diff
@@ -0,0 +1,28 @@
+from cvas.huggingface_dataclass import HuggingFaceDataset
+import torch
+from torchvision import transforms
+
+transform = transforms.Resize((32, 32))
+
+
+def test_hf_get_item():
+    hf_dataset = {
+        "train": [
+            {"img": torch.rand(3, 44, 44), "label": "dog"},
+            {"img": torch.rand(3, 96, 96), "label": "cat"},
+            {"img": torch.rand(3, 100, 100), "label": "ship"},
+            {"img": torch.rand(3, 80, 80), "label": "truck"},
+        ],
+        "test": [
+            {"img": torch.rand(3, 36, 36), "label": "airplane"},
+            {"img": torch.rand(3, 58, 58), "label": "deer"},
+        ],
+    }
```

Comment:
input dataset type is incorrect, it should one from the two types below

- from datasets import Dataset, DatasetDict

Reply:
done